muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/discussion/body/application/discussion-threads.tsx

```diff
@@ -86,9 +97,31 @@ class DDiscussionThreads extends React.Component<
     return label;
   };
 
+  /**
+   * handleSubscribeOrUnsubscribeClick
+   * @param thread thread
+   * @param isSubscribed isSubscribed
+   */
+  handleSubscribeOrUnsubscribeClick =
+    (thread: DiscussionThreadType, isSubscribed: boolean) =>
+    (e: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {
```

Comment:
Why would you name an event and not use it in any way?

Reply:
Look lines 236 and 253

Reply:
Ah, yes, I see. To prevent bubbling